lms/djangoapps/discussion/tests/test_views.py

```diff
@@ -356,11 +356,11 @@ def setUp(self):
         # course is outside the context manager that is verifying the number of queries,
         # and with split mongo, that method ends up querying disabled_xblocks (which is then
         # cached and hence not queried as part of call_single_thread).
-        (ModuleStoreEnum.Type.mongo, 1, 6, 4, 15, 3),
-        (ModuleStoreEnum.Type.mongo, 50, 6, 4, 15, 3),
+        (ModuleStoreEnum.Type.mongo, 1, 5, 3, 13, 1),
```

Comment:
@e-kolpakov: I think the purpose of these types of tests are to protect against performance impacts that would otherwise be missed, by checking that:
1. changes in the number of queries make directional sense, and
2. the numbers don't sky-rocket without us knowing it.
So, they are known to be brittle, and you typically just copy in the new results.  At least, this is my understanding.